packages/vite/src/rsc/rscWorker.ts

```diff
@@ -121,6 +125,17 @@ registerFwGlobals()
 // is already in use`.
 const dummyServer = new Server()
 
+const reactBabelConfig = getWebSideDefaultBabelConfig({
+  forVite: true,
+  forRSC: true,
+})
+reactBabelConfig.overrides.push({
+  test: /Routes.(js|tsx|jsx)$/,
+  plugins: [[RedwoodRoutesAutoLoaderRscServerPlugin, {}]],
```

Comment:
You don't need an array of arrays here, right?
```suggestion
  plugins: [RedwoodRoutesAutoLoaderRscServerPlugin],
```

Reply:
I'll double-check when the CI goes green. There was some issue with some of the jest tests failing for no reason when you didn't pass the empty object. Yeah, I'll go through the code when it passes CI to refactor and tidy.